### PR TITLE
Adding Long Descriptions

### PR DESCRIPTION
**Pull request by jerrysong1324**
Address Lux Issue #216 
Related to Lux Widget PR 67
- Added in long descriptions for actions
- Problems: some long descriptions may not be detailed enough, portions of the description will eventually be dynamically determined

### CODE REVIEW

File: lux/action/column_group.py [hunk 1]
```diff
@@ -26,6 +26,7 @@ def column_group(ldf):
     recommendation = {
         "action": "Column Groups",
         "description": "Shows charts of possible visualizations with respect to the column-wise index.",
+        "longDescription": "Shows charts of possible visualizations with respect to the column-wise index.",
```
Comment (dorisjlee) [inline lux/action/column_group.py:29]: Could we change this to `long_description` to conform with PEP8?

File: lux/action/correlation.py [hunk 2]
```diff
@@ -51,6 +51,9 @@ def correlation(ldf: LuxDataFrame, ignore_transpose: bool = True):
     recommendation = {
         "action": "Correlation",
         "description": "Show relationships between two <p class='highlight-descriptor'>quantitative</p> attributes.",
+        "longDescription": "Correlation searches through all pairwise relationship between two quantitative attributes \
+            (e.g., age, capital-gain). The visualizations are ranked from most to least linearly correlated based on \
```
Comment (dorisjlee) [inline lux/action/correlation.py:55]: `age, capital-gain` needs to be adaptive here

File: lux/action/univariate.py [hunk 3]
```diff
@@ -55,6 +55,7 @@ def univariate(ldf, *args):
         recommendation = {
             "action": "Distribution",
             "description": "Show univariate histograms of <p class='highlight-descriptor'>quantitative</p>  attributes.",
+            "longDescription": "Distribution displays univariate histogram distributions of all quantitative attributes (e.g., Horsepower, Displacement). Visualizations are ranked from most to least skewed.",
```
Comment (dorisjlee) [inline lux/action/univariate.py:58]: e.g. field needs to be adaptive

File: lux/action/univariate.py [hunk 4]
```diff
@@ -65,13 +66,15 @@ def univariate(ldf, *args):
         recommendation = {
             "action": "Occurrence",
             "description": "Show frequency of occurrence for <p class='highlight-descriptor'>categorical</p> attributes.",
+            "longDescription": "Occurence displays bar charts of counts for all categorical attributes (e.g., HighestDegree, FundingModel). Visualizations are ranked from most to least uneven across the bars. ",
```
Comment (dorisjlee) [inline lux/action/univariate.py:69]: e.g. field needs to be adaptive

**Review by dorisjlee — CHANGES_REQUESTED** [review]

**Review by cjachekang — APPROVED** [review]

**Comment by jerrysong1324** [comment]
@dorisjlee I can change the adaptive fields, but there were some other cases, where I was a bit confused with the long description. I will schedule some time to sync with you on this.

File: lux/action/correlation.py (hunk 2 above)
Comment (jerrysong1324) [reply lux/action/correlation.py:55]: resolved

File: lux/action/univariate.py (hunk 3 above)
Comment (jerrysong1324) [reply lux/action/univariate.py]: resolved

File: lux/action/univariate.py (hunk 4 above)
Comment (jerrysong1324) [reply lux/action/univariate.py]: resolved

File: lux/action/univariate.py [hunk 5]
```diff
@@ -55,6 +55,7 @@ def univariate(ldf, *args):
         recommendation = {
             "action": "Distribution",
             "description": "Show univariate histograms of <p class='highlight-descriptor'>quantitative</p>  attributes.",
+            "long_description": "Distribution displays univariate histogram distributions of all quantitative attributes (e.g., Horsepower, Displacement). Visualizations are ranked from most to least skewed.",
```
Comment (dorisjlee) [inline lux/action/univariate.py:58]: `Horsepower, Displacement` is still not adaptive.

File: lux/action/correlation.py (hunk 2 above)
Comment (dorisjlee) [reply lux/action/correlation.py:55]: I don't think this is adaptive yet?

File: lux/action/univariate.py (hunk 5 above)
Comment (jerrysong1324) [reply lux/action/univariate.py]: Whoops, fixed, my commits didn't get push for some reason.

File: lux/action/column_group.py [hunk 6]
```diff
@@ -26,6 +26,7 @@ def column_group(ldf):
     recommendation = {
         "action": "Column Groups",
         "description": "Shows charts of possible visualizations with respect to the column-wise index.",
+        "long_description": "Shows charts of possible visualizations with respect to the column-wise index.",
```
Comment (dorisjlee) [inline lux/action/column_group.py:29]: Might want to expand on this description with an image or longer description and link to the description on [this page](https://lux-api.readthedocs.io/en/latest/source/advanced/indexgroup.html).

File: lux/action/row_group.py [hunk 7]
```diff
@@ -26,6 +26,7 @@ def row_group(ldf):
     recommendation = {
         "action": "Row Groups",
         "description": "Shows charts of possible visualizations with respect to the row-wise index.",
+        "long_description": "Shows charts of possible visualizations with respect to the row-wise index.",
```
Comment (dorisjlee) [inline lux/action/row_group.py:29]: same comment as `column_group.py`

File: lux/action/column_group.py (hunk 6 above)
Comment (jerrysong1324) [reply lux/action/column_group.py]: @dorisjlee Added some descriptions (I'm not a fan of the images there, but we can chat about that more) as well as a link.

File: lux/action/row_group.py (hunk 7 above)
Comment (jerrysong1324) [reply lux/action/row_group.py]: same as above

**Review by dorisjlee — APPROVED** [review]

**Comment by dorisjlee** [comment]
Thanks @jerrysong1324 !

**Pull request merged**